github/ipld.yml

```diff
@@ -1326,7 +1326,39 @@ repositories:
   persistent-metadata:
     archived: true
   py-cid:
-    archived: true
+    advanced_security: false
+    allow_update_branch: false
+    archived: false
+    collaborators:
+      admin:
+        - dhruvbaldawa
```

Comment:
```suggestion
```

prefer to do admin through this repo rather than giving someone super powers

also I'm not sure web3-bot does anything special with Python repos, normally we rely on it to update CI scripts but I think it mainly focuses on Go and JS: https://github.com/ipdxco/unified-github-workflows?tab=readme-ov-file#availability-and-future-directions

so I think you could just remove the entire `collaborators` block and rely on teams